app/views/veterans/profile/_mentor.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if @mentor.nil? %>
+  <center><i>You have not requested a mentor</i></center>
```

Comment:
Per [i's](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i#Notes) documentation, use newer tags or CSS to italicize the text. 

[center](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center) has been deprecated.